### ISSUE TITLE
[Manual] Mention that Hall Effect PCB is for legacy support and not in any way an upgrade

### ISSUE THREAD

**Issue by ChaosBlades** [VoronDesign/Voron-2#388]
In the manual the Hall Effect is presented as an upgrade. At least I interpreted it that way and it seems several people in the Amazon reviews for the Hall Effect PCB I bought thought this as well. Turns out the Hall Effect sensor is worse in basically every way from the micro switches as I found out asking about it in Discord. This should really be pointed out in the documentation or just removed from the manual all together.

**Comment by raymondh2** [comment]
I'll bring it up to the team.

**Comment by nemgrea** [comment]
> In the manual the Hall Effect is presented as an upgrade. At least I interpreted it that way and it seems several people in the Amazon reviews for the Hall Effect PCB I bought thought this as well. 


![unknown](https://user-images.githubusercontent.com/52015997/191835261-d392eeac-485a-4c17-8a12-0a934f68d21c.png)

its shown as an option, i dont think there is anything in here that expresses that one is better than the other or that one would be an upgrade over the other, simply two options are shown.

**Comment by ChaosBlades** [comment]
I don't think that is good enough. According to my discussion on Discord the hall effect isn't any more reliable than micro switches, it is sensitive to chamber temp, switching to PLA when tuned for ABS it could go so out of spec enough to cause a crash, it drifts over time, and you can't even increase home speed. It is not an improvement in any way.

What I was told was...

> StevenB V2.3370 — Yesterday at 2:55 AM
Because it was a test that failed and it's still kept as an option because of that.

When I go to some of these selling on Amazon I see people complaining that the Hall Effect PCBs are using a different connector (Molex instead of JST) so they need to re-cable when moving from Microswitch. Which is a downgrade. So clearly myself and other have fallen victim to this simply being show as an option. I am sure there is some kind of lesson here on assuming things.

**Comment by nemgrea** [comment]
i agree is is not any more reliable at all NOR is it ever claimed to be more reliable, im not sure why you are assuming that it would be more reliable when it is not presented as "better" or of higher quality in any of the documentation. it is simply showing steps for an OPTION that exists and happens to be popular. no claims of benefits or performance are shown, stated, or implied in the manual...

**Comment by subatomicglue** [comment]
I wonder if OP is getting at:
 - Shouldn't the default of voron's manual be to clearly highlight the "best", for the optimal voron experience?   Like, If I'm presented with 2 options, and I know not which is better, shouldn't there be something helping guide me to choose, so that I make the best voron printer available and not go down "less-good" paths?

**Comment by raymondh2** [comment]
Our BOM already lists the normal microswitch setup and not any of the PCBs. But these PCBs are included in the manual because they are so common and we get too many questions when we don't include the instructions for it.

**Comment by asdasdwa-v02679** [comment]
Could we possibly solve this problem by adding text on page 191 below the images that has a warning specifying that although the pcbs can simplify wiring they tend to introduce their own issues with repeatability and aren't recommended.  

or is it just the hall effect pcb? if so perhaps it would read more like "although the hall effect sensor pcbs have become super common and can simplify wiring, they tend to introduce their own issues with repeatability and aren't recommended."

does this seem like a good solution according to yall? if so and people approve I am happy to make the edit to the page :)

**Comment by majarspeed** [comment]
This is not the only place in the build where you can make a choice. These options exist because people have used them successfully. But to try to critique the benefits of one over the other is not really the purpose of an assembly manual.

**Comment by asdasdwa-v02679** [comment]
@majarspeed  I understand that but It seems that Nemgrea marked it as manual so I was trying to find a solution. also It does seem a fair few people get confused by this page and so a edit might be in order ?

**Comment by hansonap** [comment]
I believe there is a disconnect between the issue and the response to the proposed solution. The OP saw that there was multiple ways to accomplish the same task and was confused which option should be selected. Then in an effort to contribute to the project identified the manual as needing to be updated. I think making this suggestion was fair as there is no documentation on the ethos of the manual, in fact this repo appears to lack any contribution guidelines.

This specific instance underpins a much larger underlining issue - sub-system/mod **selection** is undocumented. [Voron-Hardware](https://github.com/VoronDesign/Voron-Hardware/tree/master) lists mods but most poorly described and lack specifications (what is the accuracy and precision of the hall-effect option, what are the environmental limitations, etc.).

3 years later I had the exact same issue as the OP regarding the end stops, and this is the first information I found regarding if the hall effect option was better. And now I have to go do the same for the z-probe. Is Tap better, is there something that is better than Tap now that it is 2026?

I am very interested in the Voron project to fit my need for building a cost effective large format printer that supports multiple tool heads, and as it is 2026 (and viewing it through that lens) I'm finding the information incomplete and fragmented, all of which detracts from the value proposition (and relevance) of the Voron project.

I have two immediate recommendations: 1. identify a solution to the OPs issue (lack of information for selection of x/y homing) and 2. Creation of a new issue to develop solutions for the underlining issues (lack of mod/option specifications, mod/option selection, fragmented and incomplete information). To summarize in another way, as someone who is new to Voron, it appears as you are supposed to build it to 2022 spec and then spend the rest of your life's free time searching the web and tinkering with mods to approximate what you know a new built Voron of today should be.
